config-model/src/main/java/com/yahoo/vespa/model/search/NodeResourcesTuning.java

```diff
@@ -121,20 +120,17 @@ private void tuneRequestThreads(ProtonConfig.Builder builder) {
         builder.numthreadspersearch(threadsPerSearch);
     }
 
-    private void tuneWriteFilter(ProtonConfig.Writefilter.Builder builder) {
-        // "Reserve" 1GB of memory for other processes running on the content node (config-proxy, cluster-controller, metrics-proxy)
-        double reservedMemoryGb = 1;
-        double defaultMemoryLimit = new ProtonConfig.Writefilter(new ProtonConfig.Writefilter.Builder()).memorylimit();
-        double scaledMemoryLimit = ((usableMemoryGb() - reservedMemoryGb) * defaultMemoryLimit) / usableMemoryGb();
-        builder.memorylimit(scaledMemoryLimit);
-    }
-
     /** Returns the memory we can expect will be available for the content node processes */
     private double usableMemoryGb() {
-        if ( ! combined ) return resources.memoryGb();
+        // "Reserve" 1GB of memory for other processes running on the content node (config-proxy, cluster-controller, metrics-proxy).
+        final double reservedMemoryGb = 1;
```

Comment:
Instead of spread '1' around I would make it a constant that could be used in the tests. Or a static method that could be used in the tests compute real memory.

Reply:
Yes, I agree. Fixed. @baldersheim PTAL.